tket/test/src/test_CompilerPass.cpp

```diff
@@ -2026,5 +2026,17 @@ SCENARIO(
     REQUIRE(cu.get_circ_ref().n_gates() == 1);
   }
 }
+
+SCENARIO("PauliExponentials") {
+  GIVEN("A PhasedX gate") {
+    // https://github.com/CQCL/tket/issues/1244
+    Circuit c(1);
+    c.add_op<unsigned>(OpType::PhasedX, {0.5, 0.6}, {0});
```

Comment:
Just for completeness, we could add another gate to test for a Clifford angle of beta (just switch the angles). Otherwise this all looks good to me!